### DISCUSSION TITLE
为啥我的command只对 控制台有效啊？ 解析qq信息必须EventChannel吗？

### DISCUSSION THREAD

**Discussion by BohrX** [mamoe/mirai-console#289] [展示和讲述]
rt 我写demo的时候发现自己的SimpleCommand只有在使用 控制台输入时才有效 qq信息无效。必须要通过 EventChannel 写自己的解析才行吗 我看Mirai Console  的教程上有说可以解析啊
    // 在控制台执行 "/manage <群号>.<群员> <持续时间>",
    // 或在聊天群内发送 "/manage <@一个群员> <持续时间>",
    // 或在聊天群内发送 "/manage <目标群员的群名> <持续时间>",
    // 或在聊天群内发送 "/manage <目标群员的账号> <持续时间>"
    // 时调用这个函数

**Comment by AdorableParker** [comment]
请看[`chat-command`](https://github.com/project-mirai/chat-command)项目，以及[`mirai-console更新日志`](https://github.com/mamoe/mirai-console/releases/tag/1.0.0)

**Reply by BohrX** [reply]
我在mcl里装了chat-command ，他自动安装只能装到 0.4 版本，所以我手动装了最新的0.51。 结果都没用啊！

**Reply by AdorableParker** [reply]
装了`chat-command`的情况下，请检查你是否给予了相应权限
